torch/onnx/utils.py

```diff
@@ -84,13 +84,16 @@ def export(model, args, f, export_params=True, verbose=False, training=None,
             operator_export_type = OperatorExportTypes.ONNX_ATEN_FALLBACK
         else:
             operator_export_type = OperatorExportTypes.ONNX
+    if enable_onnx_checker is not None:
+        warnings.warn("`enable_onnx_checker' is DEPRECATED. Please stop using it."
```

Comment:
Should be something like `enable_onnx_checker arg is deprecated and ignored. It will be removed in the next release.`